net/tor-hs/files/nextcloud-update.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+# This script disables trusted domain check for given
+# hidden service address
+
+NEXTCLOUD_CLI_SCRIPT="/srv/www/nextcloud/occ"
+
+nextcloud_cli() {
+	sudo -u nobody php-cli "$NEXTCLOUD_CLI_SCRIPT" "$@"
+}
+
+
+nextcloud_add_domain() {
+	onion="$1"
+	if [ ! -z "$onion" ]; then
+		if nextcloud_cli config:system:get trusted_domains |grep "$onion" ; then
+			echo "Info: Trusted domains already disabled. Nothing to do." >&2
+		else
+
+			echo "Info: Disabling trusted domains." >&2
+			nextcloud_cli config:system:set trusted_domains 1000 --value=$onion
+		fi
+	fi
+}
+
+print_help() {
+	echo "Help"
+}
+
+if [ ! -f "$NEXTCLOUD_CLI_SCRIPT" ]; then
```

Comment:
I assume this can be replaced with [ -f ] || {  }

Reply:
Fixed.